frontend/test/__support__/e2e/test_tables_data.js

```diff
@@ -0,0 +1,54 @@
+export const many_data_types_data = [
+  {
+    uuid: "a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11",
+    integer: 1,
+    integerUnsigned: 2,
```

Comment:
What do you think about including the word "rows" in the name here? It could be me, but I didn't immediately catch that these array is going to be sent to `insert`